tools/eslint-rules/avoid-prototype-pollution.js

```diff
@@ -64,8 +66,10 @@ function checkPropertyDescriptor(context, node) {
 }
 
 function createUnsafeStringMethodReport(context, name, lookedUpProperty) {
+  const lastDotPosition = '$String.prototype.'.length;
+  const unsafePrimordialName = `StringPrototype${name.charAt(lastDotPosition).toUpperCase()}${name.slice(lastDotPosition + 1, -1)}`;
   return {
-    [`${CallExpression}[expression.callee.name=${JSON.stringify(name)}]`](node) {
+    [CallExpression(unsafePrimordialName)](node) {
```

Comment:
```suggestion
    [`${CallExpression}${unsafePrimordialName}`](node) {
```
ins't `CallExpression` a string? am I missing something?

Reply:
NMV, looked at the old version